src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,20 @@
-export const Tabs = () => {};
+export const Tabs = ({ tab, select, set }) => {
+  const check = tab.title === select;
+  let condition = '';
+
+  if (check) {
+    condition = 'go';
+  }
+
+  return (
+    <li data-cy="Tab" className={condition && 'is-active'}>
+      <a
+        href={`#${tab.id}`}
+        data-cy="TabLink"
+        onClick={e => set(e.target.textContent)}
+      >
+        {tab.title}
+      </a>
+    </li>
```

Comment:
When rendering lists in React, it's crucial to provide a `key` prop to each child in the list. The `key` prop helps React identify which items have changed, are added, or are removed and should be a string that uniquely identifies each list item. In this case, you could use `tab.id` as a key: `<li key={tab.id} ...>...</li>`